### PR TITLE
chore: update README to use autosynth

### PR DESCRIPTION
**Pull request by stephaniewang526**

### CODE REVIEW

File: .readme-partials.yaml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+  ```java
+  // Create a dataset
+  String datasetId = "my_dataset_id";
+  bigquery.create(DatasetInfo.newBuilder(datasetId).build());
+  ```
+
+  #### Creating a table
+  With BigQuery you can create different types of tables: normal tables with an associated schema,
+  external tables backed by data stored on [Google Cloud Storage][cloud-storage] and view tables that
```
Comment (chingor13) [inline .readme-partials.yaml:19]: You'll need to make this link a non-reference link as we don't have it in the template.

File: README.md [hunk 2]
```diff
@@ -50,57 +57,35 @@ libraryDependencies += "com.google.cloud" % "google-cloud-bigquery" % "1.111.1"
...
-[Application Default Credentials](https://developers.google.com/identity/protocols/application-default-credentials).
-These credentials are automatically inferred from your environment, so you only need the following
-code to create your service object:
+## About Cloud BigQuery
 
-```java
-import com.google.cloud.bigquery.BigQuery;
-import com.google.cloud.bigquery.BigQueryOptions;
 
-BigQuery bigquery = BigQueryOptions.getDefaultInstance().getService();
```
Comment (chingor13) [inline README.md:99]: Note: this snippet didn't get moved to the custom content, but is may not actually be useful

Comment (stephaniewang526) [reply README.md:99]: Yeah i think we can do without this snippet.

File: .readme-partials.yaml (hunk 1 above)
Comment (stephaniewang526) [reply .readme-partials.yaml:19]: Done

Comment (chingor13) [reply .readme-partials.yaml:19]: I don't see a change here. Any change you manually make in the README.md itself will be wiped on the next regeneration.

Comment (stephaniewang526) [reply .readme-partials.yaml]: PTAL again

**Pull request merged**